Three/include/CGAL/Three/TextRenderer.h

```diff
@@ -60,7 +60,7 @@ public :
         :x(p_x), y(p_y), z(p_z),_3D(p_3D), _is_always_visible(always_visible), m_text(p_text), m_font(font), m_color(p_color)
     {
        QFontMetrics fm(m_font);
-       _width = float(fm.width(m_text));
+       _width = float(fm.width(m_text)+2);
```

Comment:
`QFontMetric::width` is obsoleted: https://doc.qt.io/qt-5/qfontmetrics-obsolete.html#width